tests/Language/Haskell/Stylish/Step/Imports/Tests.hs

```diff
@@ -930,3 +1009,248 @@ case38 = assertSnippet (step (Just 80) $ fromImportAlign File)
     [ "import Happstack.Server"
     , "import HSP"
     ]
+
+--------------------------------------------------------------------------------
+case39 :: Assertion
+case39 = assertSnippet (step Nothing options)
+  [ "import Something.A"
+  , "import SomethingElse.A"
+  , "import SomeThing.B"
+  , "import SomeThingelse.B"
+  ]
+  [ "import           Something.A"
+  , "import           SomeThing.B"
+  , ""
+  , "import           SomethingElse.A"
+  , "import           SomeThingelse.B"
```

Comment:
You know, I don't know how I feel about this. Module names are case-sensitive in Haskell, so why would I group these together?

Reply:
Yeah, not sure. I saw that the import logic for sorting was case-insensitive, so I wanted the grouping logic to match that. Might be better not to do that at all though.